### ISSUE TITLE
Polars support

### ISSUE THREAD

**Issue by axelwas** [ClickHouse/clickhouse-connect#111]
Hello, thank you for this great library!
Is it planned to support [Polars](https://www.pola.rs/) library at some point ?

**Comment by genzgd** [comment]
Polars is not currently on the roadmap, but we do look at the community interest in new features on a regular basis and adjust our priorities accordingly.  Of course we'd welcome a pull request from someone who is familiar with Polars.

**Comment by qqletter** [comment]
Yes, I do hope this client will support polars.

**Comment by NotSimone** [comment]
You can query to get pyarrow tables, which can then zero-copy into Polars with something like this:
```
df = pl.from_arrow(client.query_arrow(query))
```

**Comment by genzgd** [comment]
Feedback based on @NotSimone's comment would be appreciated -- is using PyArrow enough of a solution or does Polars use some other internal data format?

**Comment by axelwas** [comment]
Polars does use Pyarrow under the hood so I think this would work. 

Since my question, we ended up sticking with Pandas. Would it be possible to optionally make `query_df` return a Pyarrow backed Pandas DataFrame instead of Numpy backed  ? I see `use_numpy` param is hardcoded to `true` [here](https://github.com/ClickHouse/clickhouse-connect/blob/8cc27b24bd9071c7f303143969366d7aad954c2a/clickhouse_connect/driver/client.py#L332).

**Comment by genzgd** [comment]
Agreed that PyArrow DataFrames would be useful - https://github.com/ClickHouse/clickhouse-connect/issues/326.   However I'm sure when it will it will be prioritized.
